CraftingRevisions/Patches/GameManager_Awake.cs

```diff
@@ -11,11 +11,11 @@ internal class Panel_Crafting_Initialize
 		private static void Postfix()
 		{
 			// call TLD LoadAllUserBlueprints
+#warning BlueprintManager.LoadAllUserBlueprints is not called by the game itself yet, we may need to remove this to prevent dupe recipes in the future.
```

Comment:
It won't cause duplicate recipes. User recipes are cleared whenever this is called.